tests/Microsoft.Identity.Test.Unit/PublicApiTests/LoggerTests.cs

```diff
@@ -42,20 +42,25 @@ public void ConstructorComponentTest()
         }
 
         [TestMethod()]
-        public void CallbackTestErrorTest()
+        public void CallbackTestLogAlwaysTest()
```

Comment:
Could some of these test methods be combined into parametrized tests?

Reply:
Yes, looking into it

Reply:
updated